### DISCUSSION TITLE
how to create"mimic_derived.bg"

### DISCUSSION THREAD

**Discussion by Joywanglulu** [MIT-LCP/mimic-code#1110] [MIMIC-IV]
While extracting comorbidity patient via "mimic-code/mimic-iv/concepts/comorbidity/charlson.sql", an error occurred  " relation "mimic_derived.bg" does not exist". I also try to run first_day_bg, but it also required mimic_derived.bg. Thank you so much for your help!

**Comment by briangow** [comment]
@Joywanglulu there is a mimic_derived table on BigQuery. The charlson.sql script was designed to be run on BigQuery. Are you using BigQuery? Do you see the table? When I run the script there it works as expected.
